js/gamestate.js

```diff
@@ -221,17 +408,24 @@ class GameState {
     for (let key in card) {
       copy[key] = card[key]
     }
-
+    copy.canAct = false; 
     this.current().hand.push(copy)
   }
 
   beginTurn() {
     this.current().maxMana = Math.min(1 + this.current().maxMana, 10)
+    if (DEBUG) {
```

Comment:
in general i would say, if there is some handy functionality for debugging, but it's not documented and only one person would use it, leave it out of master. if you want it in master then make it work normally instead of having it be a const like DEBUG